pkg/logging/logging.go

```diff
@@ -177,10 +177,11 @@ func getContainerWait(ctx context.Context, hostAddress string, config *logging.C
 	// If task was not found, it's possible that the container runtime is still being created.
 	// Retry every 100ms.
 	ticker := time.NewTicker(100 * time.Millisecond)
+	defer ticker.Stop()
```

Comment:
Probably this should be a separate commit

Reply:
fixed